src/Routes/Addcom.jsx

```diff
@@ -0,0 +1,34 @@
+import { postComment } from "../../utils";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+const AddCom = () => {
+  const [newComment, setNewComment] = useState("");
+  const { article_id } = useParams();
+  const [confirmationMessage, setConfirmationMessage] = useState("");
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(newComment);
+    postComment(article_id, newComment).then(() => {
+      setConfirmationMessage(`Your Comment "${newComment}" has been added`);
+    });
+  };
+
+  return (
+    <>
+      <form onSubmit={handleSubmit}>
+        <label></label>
+        <input
```

Comment:
This should not be an input, we could look if we have tags for larger text areas.
The field should also be reset after submission 🙂